spec/resolver/platform_spec.rb

```diff
@@ -90,11 +165,11 @@
       end
     end
 
-    it "reports on the conflict" do
+    it "takes the ruby version as fallback" do
       platforms "ruby", "java"
       dep "foo"
 
-      should_conflict_on "baz"
+      should_resolve_as %w[bar-1.0.0 baz-1.0.0 foo-1.0.0]
```

Comment:
This should be resolved instead of conflicting.

See also:

* https://github.com/rubygems/bundler/pull/7522#issuecomment-572635007
* https://github.com/rubygems/bundler/pull/7522#issuecomment-572882729

Reply:
Nice, and I remember that during my investigation I was getting a correct resolution, but with one spec being duplicated, and I see that you fixed that with your `uniq` addition :tada:.